modules/auxiliary/admin/dns/dyn_dns_update.rb

```diff
@@ -36,6 +36,7 @@ def initialize
     register_options([
       OptString.new('DOMAIN', [true, 'The domain name']),
       OptAddress.new('RHOST', [true, 'The vulnerable DNS server IP address']),
+      OptPort.new('RPORT', [true, "DNS server port", 53]),
```

Comment:
Looks like there's a call to deregister the RPORT option later in the file, which would explain why `RPORT` wasn't showing up in the options:

```
deregister_options('RPORT')
```

Reply:
That would do it. Why didn't I spot that!